game.go

```diff
@@ -303,28 +300,42 @@ type costumeConfig struct {
 	Y                float64 `json:"y"`
 	FaceRight        float64 `json:"faceRight"` // turn face to right
 	BitmapResolution int     `json:"bitmapResolution"`
+	Mode             string  `json:"mode"`
+}
+
+func toSceneMode(mode string) int {
+	switch mode {
+	case "repeat":
+		return sceneModeRepeat
+	case "fillCut":
+		return sceneModeFillCut
+	case "fillRatio":
+		return sceneModeFillRatio
+	}
+	return sceneModeFill
 }
 
 type cameraConfig struct {
 	On string `json:"on"`
 }
 
 type mapConfig struct {
-	Mode   string `json:"mode"`
-	Width  int    `json:"width"`
-	Height int    `json:"height"`
+	Behavior string  `json:"behavior"`
+	StepUnit float64 `json:"stepUnit"`
```

Comment:
maybe just call it `unit`? it's shared by `sprite.Move`, `sprite.Step` and `sprite.DistanceTo`, even `sprite.SetXpos`, `sprite.SetYpos`, `sprite.SetXYpos`, etc. (if possible), not only `sprite.Step`.